layers/timeline_semaphore.c

```diff
@@ -888,6 +888,7 @@ static VkResult timeline_wait_locked(struct device_data *device,
             int ret = pthread_cond_timedwait(&device->queue_submit,
                                              &device->lock, &abstime);
             assert(ret != EINVAL);
+            (void)ret;
```

Comment:
This resolves an unused-variable warning in a release build:

../third_party/vulkan-extensionlayer/layers/timeline_semaphore.c:888:17: warning: unused variable ‘ret’ [-Wunused-variable]
             int ret = pthread_cond_timedwait(&device->queue_submit,
                 ^~~